daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1140,6 +1194,17 @@ message Update {
     Expr key = 2;
   }
 
+  // Try-catch block.
+  message TryCatch {
+    Type return_type = 1;
+    Expr try = 2;
+
+    // Name of the bound expression variable.
+    // *Must be a valid interned identifier*
+    int32 var_interned_str = 3;
+    Expr catch = 4;
```

Comment:
Do not really like `catch` here neither. I do not find any really better name neither. maybe `catcher`